etc/rucio.cfg.template

```diff
@@ -29,6 +29,7 @@ protocol_stat_retries = 6
 
 [core]
 geoip_licence_key = LICENCEKEYGOESHERE  # Get a free licence key at https://www.maxmind.com/en/geolite2/signup
+default_mail_from = atlas-adc-ddm-support@cern.ch
```

Comment:
Please change to spamspamspam@cern.ch so nobody gets the wrong idea :-)

Reply:
should I change all occurrences?

Reply:
Yes please.